Boot2Docker Status/AppDelegate.m

```diff
@@ -172,11 +180,24 @@ - (void)stopBoot2Docker {
     
     [task setTerminationHandler:^(NSTask *t) {
         _haveRequestedStateChange = NO;
-        _statusItem.image = [NSImage imageNamed:@"docker-alt"];
+		_statusItem.image = [self _iconWithNotificationBubbleWithColor:[NSColor redColor]];
     }];
     
     [task launch];
 }
 
+- (NSImage*)_iconWithNotificationBubbleWithColor:(NSColor*)color {
+	NSImage* image = [(_darkMode ? [NSImage imageNamed:@"docker-dark"] : [NSImage imageNamed:@"docker"]) copy];
+
```

Comment:
Add this to use the alternative images when boot2docker is down (for colorblind users)
```objectivec
if(![self isBoot2DockerRunning]) {
        image = [(_darkMode ? [NSImage imageNamed:@"docker-alt-dark"] : [NSImage imageNamed:@"docker-alt"]) copy];
}
```